service/api/handler.go

```diff
@@ -346,8 +346,11 @@ func (h *handler) processError(c *gin.Context, resp *errors.ErrorAndStatus) {
 	c.JSON(resp.StatusCode, resp.Error)
 }
 
-func (h *handler) toJson(req any) string {
+func (h *handler) toJsonLogging(req any) string {
 	str, err := json.Marshal(req)
+	if len(str) > 1000 {
+		str = str[0:1000]
+	}
```

Comment:
I thought this logging went to Datadog, is there a specific reason why we need to limit its size? I don't see a reason to limit this if it's not going to Temporal

Reply:
This is for the request object. Logging will be too expensive too if the req is too big (>100 bytes).

This one is actually not going to Datadog because we have another internal handler (see `handler.go` in internal repo). 

I wanted to limit this otherwise my local dev will look pretty bad when sending the 1MB request.